src/atopile/front_end.py

```diff
@@ -224,6 +224,10 @@ class NOTHING:
     """A sentinel object to represent a "nothing" return value."""
 
 
+class SkipPriorFailedException(Exception):
```

Comment:
This exception helps us cut off execution of code-paths which fail because they rely on bit that have already failed